packages/ui-extensions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/ui-extensions",
-  "version": "2025.10.0",
+  "version": "2025.10.0-rc.0",
```

Comment:
I ran a `changesets version` locally when this was `2025.10.0` and it'd create a `2025.11.0.rc-0` version. 

Changing this here to `2025.10.0-rc.0` before running `changeset pre enter rc` made `changeset version` change it to `2025.10.0-rc.1` which is what we would want.